src/lib/components/commons/SearchBar/SearchBar.tsx

```diff
@@ -0,0 +1,38 @@
+import {
+  CURRENT_COLOR_CLASS_NAME,
+  POINTER_CURSOR_CLASS_NAME,
+} from '../../icons/SvgIcon';
+import {
+  StyledSearchBar,
+  StyledSearchInput,
+  StyledSearchIcon,
+} from './SearchBar.styled';
+import { ISearchBarProps } from './SearchBar.types';
+
+export const SearchBar = (props: ISearchBarProps) => {
+  const { setSearchValue } = props;
+  return (
+    <>
+      <StyledSearchBar>
+        <StyledSearchInput
+          name='search'
+          placeholder='חפש...'
```

Comment:
if you use onChange , you must make it controlled , and pass the value as well...

Reply:
What do you mean by controlled?